sentry_sdk/integrations/starlette.py

```diff
@@ -65,9 +65,14 @@
 
 try:
     # Optional dependency of Starlette to parse form data.
-    import multipart  # type: ignore
+    try:
+        # python-multipart 0.0.13 and later
+        import python_multipart  # type: ignore
+    except ImportError:
+        # python-multipart 0.0.12 and later
```

Comment:
Did you mean "earlier" here?

```suggestion
        # python-multipart 0.0.12 and earlier
```

Reply:
Good catch. Thanks. I will fix this.